includes/OutputPage.php

```diff
@@ -2127,14 +2127,14 @@ public function output() {
 
 			// Hook that allows last minute changes to the output page, e.g.
 			// adding of CSS or Javascript by extensions.
-			wfRunHooks( 'BeforePageDisplay', array( &$this, &$sk ) );
+			wfRunHooks( 'BeforePageDisplay', array( $this, &$sk ) );
 
 			wfProfileIn( 'Output-skin' );
 			$sk->outputPage();
 			wfProfileOut( 'Output-skin' );
 		}
 
-		wfRunHooks( 'BeforeSendCacheControl', array( &$this ) ); // Wikia change
+		wfRunHooks( 'BeforeSendCacheControl', array( $this ) ); // Wikia change
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)